### PR TITLE
emitClient player array support

### PR DESCRIPTION
**Pull request by tastydev**
example:

```
//sent to all players inside array
alt.emitClient([player, player, player], 'client:testEvent', "test5", 4);
//sent to specific player
alt.emitClient(player, 'client:testEvent', "test5", 4);
//sent to all players
alt.emitClient(null, 'client:testEvent', "test5", 4);
```

### CODE REVIEW

File: src/bindings/Main.cpp [hunk 1]
```diff
@@ -71,23 +71,44 @@ static void EmitClient(const v8::FunctionCallbackInfo<v8::Value>& info)
...
+	
+	Ref<IPlayer> singlePlayer;
+
+	if (info[0]->IsNull()) {
+		//if first argument is null this event gets send to every player
+		ICore::Instance().TriggerClientEvent(singlePlayer, eventName.ToString(), mvArgs);
+		return;
+	}
+	
+	if (!info[0]->IsNull() && info[0]->IsArray())
```
Comment (FabianTerhorst) [inline src/bindings/Main.cpp:88]: you sure the IsNull is needed? Because when its a array it can't really be Null.

Comment (tastydev) [reply src/bindings/Main.cpp:88]: i did it to be save but i can actually remove it cause the if on top of that if checks for isNull() already

**Comment by tastydev** [comment]
altmp/altv-issues#948

File: src/bindings/Main.cpp (hunk 1 above)
Comment (LeonMrBonnie) [reply src/bindings/Main.cpp]: The `Is*` calls only ever return true for one of them, so the `IsNull` is/was unnecessary here

**Review by LeonMrBonnie — CHANGES_REQUESTED** [review]
Please use the brace style as in the rest of the module, so braces always go on their own line

**Review by LeonMrBonnie — COMMENTED** [review]
Did you test if this works?

**Comment by tastydev** [comment]
yes it works without any issues

**Review by LeonMrBonnie — APPROVED** [review]

**Pull request merged**